v2/src/ctrlfbe/serializers.py

```diff
@@ -98,50 +117,47 @@ def create(self, validated_data):
 
 
 class PageDetailSerializer(serializers.ModelSerializer):
-    id = serializers.SerializerMethodField(method_name="get_id")
-    topic = serializers.SerializerMethodField(method_name="get_topic")
-    owners = serializers.SerializerMethodField(method_name="get_owners")
-    issue_id = serializers.SerializerMethodField(method_name="get_issue_id")
-
     class Meta:
-        model = PageHistory
-        exclude = ["owner", "page"]
-        read_only_fields = ["id", "created_at"]
-
-    def get_id(self, page_history):
-        temp = page_history.page.id
-        return temp
-
-    def get_topic(self, page_history):
-        temp = page_history.page.topic.id
-        return temp
-
-    def get_owners(self, page_history):
-        page = page_history.page
-        owners = list(page.owners.values_list("id", flat=True))
-        return owners
+        model = Page
 
-    def get_issue_id(self, page_history):
+    def to_representation(self, page):
+        version_no = self.context["version_no"]
+        page_history_queryset = page.page_history.filter(page=page, version_no=version_no)
```

Comment:
queryset을 변수 명에 나타내고 싶었던 의도라면, 타입힌트를 써주면 더 좋을 것 같음~

```suggestion
        page_history: Queryset[PageHistory] = page.page_history.filter(page=page, version_no=version_no)
```